.travis.yml

```diff
@@ -1,12 +1,15 @@
-language: objective-c
+language: csharp
 
-env:
-  matrix:
-    - MONO_VERSION="3.10.0"
+os:
+  - linux
 
-install:
-  - wget "http://download.mono-project.com/archive/${MONO_VERSION}/macos-10-x86/MonoFramework-MDK-${MONO_VERSION}.macos10.xamarin.x86.pkg"
-  - sudo installer -pkg "MonoFramework-MDK-${MONO_VERSION}.macos10.xamarin.x86.pkg" -target /
+mono:
+  - latest
 
-script: 
+sudo: false  # use the new container-based Travis infrastructure
+
+before_install:
```

Comment:
You can change the permission in the github repo http://stackoverflow.com/questions/13454764/how-to-chmod-0777-a-file-and-commit-as-is-to-git-on-windows